03 ColorPicker/readme.md

```diff
@@ -75,8 +75,7 @@ Install [Node.js and npm](https://nodejs.org/en/) (v6.6.0 or newer) if they are
   ### ./src/reducers/userProfile.ts
 ```diff
   import {actionsEnums} from '../common/actionsEnums';
-  import {updateUserProfileName} from '../actions/updateUserProfileName';
-+ import { Color } from "../model/color";
+  + import { Color } from "../model/color";
```

Comment:
The `+` symbol has to be on the left without previous spaces because it's needed to apply green style

Reply:
Should be fixed now. I hope. 🤞

Reply:
Not yet, the previous line is ok

Reply:
Not sure what you mean. Do you mean the `import {updateUserProfileName} from '../actions/updateUserProfileName';` line?

That line is not needed because "updateUserProfileName" is not used anywhere in userProfile.ts. I will double check again when I am back at work tomorrow, but it that line seemed unnecessary.

Reply:
Sorry, I mean the line [79](https://github.com/Lemoncode/redux-by-sample/pull/51#discussion-diff-143650942L79) in PR (the red one) is ok. Because the `+` symbol is on the far left